src/garage/torch/algos/ppo.py

```diff
@@ -75,19 +80,24 @@ def __init__(self,
                          policy_ent_coeff=policy_ent_coeff,
                          use_softplus_entropy=use_softplus_entropy,
                          stop_entropy_gradient=stop_entropy_gradient,
-                         entropy_method=entropy_method)
+                         entropy_method=entropy_method,
+                         minibatch_size=minibatch_size,
+                         max_optimization_epochs=max_optimization_epochs)
 
         self._lr_clip_range = lr_clip_range
 
-    def _compute_objective(self, advantages, valids, obs, actions, rewards):
-        """Compute objective using surrogate value and clipped surrogate value.
+    def _compute_objective(self, advantages, obs, actions, rewards):
+        """Compute objective value.
 
         Args:
-            advantages (torch.Tensor): Expected rewards over the actions
-            valids (list[int]): length of the valid values for each path
+            advantages (torch.Tensor): Advantage value at each step.
+                with shape :math:`(T, )`.
             obs (torch.Tensor): Observation from the environment.
-            actions (torch.Tensor): Predicted action.
-            rewards (torch.Tensor): Feedback from the environment.
+                with shape :math:`(T, observation_dim)`.
```

Comment:
please use `A*` for unknown action shapes and `O*` for unknown observation shapes, per the style in https://github.com/rlworkgroup/garage/blob/master/src/garage/_dtypes.py